File: backend/schemas/enums/enums.go
package enums

type Frequency string

const (
	Daily          Frequency = "daily"
	MondayToFriday Frequency = "mondayToFriday"
	Weekly         Frequency = "weekly"
	Monthly        Frequency = "monthly"
	Yearly         Frequency = "yearly"
	Unique         Frequency = "unique"
)

type Priority string

const (
	Essential Priority = "essential"
	Normal    Priority = "normal"
	Low       Priority = "low"
)

type MuscleHead string

const (
	HeadLong     MuscleHead = "Long"
	HeadShort    MuscleHead = "Short"
	HeadMiddle   MuscleHead = "Middle"
	HeadSuperior MuscleHead = "Superior"
	HeadInferior MuscleHead = "Inferior"
)

type MuscleFunction string

const (
	Target     MuscleFunction = "Target"
	Synergist  MuscleFunction = "Synergist"
	Stabilizer MuscleFunction = "Stabilizer"
)

type CategoryType string

const (
	Tasks    CategoryType = "tasks"
	Finances CategoryType = "finances"
)

type ItemTag string

const (
	Supermarket ItemTag = "supermarket"
	Drugstore   ItemTag = "drugstore"
	Supplements ItemTag = "supplements"
	Wish        ItemTag = "wish"
)

File: app/lib/models/enums.dart
enum Priority {
  essential,
  normal,
  low
}

extension PriorityExtension on Priority {
  String get label {
    switch (this) {
      case Priority.essential:
        return 'essential';
      case Priority.normal:
        return 'normal';
      case Priority.low:
        return 'low';
      }
  }
}

enum FrequencyTransaction {
  weekly,
  monthly,
  yearly,
  unique
}


extension FrequencyTransactionExtension on FrequencyTransaction {
  String get label {
    switch (this) {
      case FrequencyTransaction.weekly:
        return 'weekly';
      case FrequencyTransaction.monthly:
        return 'monthly';
      case FrequencyTransaction.yearly:
        return 'yearly';
      case FrequencyTransaction.unique:
        return 'unique';

    }
  }
}

enum Frequency {
  daily,
  mondayToFriday,
  weekly,
  monthly,
  yearly,
  unique
}

extension FrequencyExtension on Frequency {
  String get label {
    switch (this) {
      case Frequency.daily:
        return 'daily';
      case Frequency.mondayToFriday:
        return 'mondayToFriday';
      case Frequency.weekly:
        return 'weekly';
      case Frequency.monthly:
        return 'monthly';
      case Frequency.yearly:
        return 'yearly';
      case Frequency.unique:
        return 'unique';
    }
  }
}